gincmd/common.go

```diff
@@ -247,42 +247,20 @@ func verboseOutput(statuschan <-chan git.RepoFileStatus) (filesuccess map[string
 	filesuccess = make(map[string]bool)
 	var ro string
 	var tmprawin, tmpfname string
-	type Act struct {
-		Command string `json:"command"`
-		Note    string `json:"note"`
-		Key     string `json:"key"`
-		File    string `json:"file"`
-	}
-	type Jsonout struct {
-		ByteProgress    int    `json:"byte-progress"`
-		Action          Act    `json:"action"`
-		TotalSize       int    `json:"total-size"`
-		PercentProgress string `json:"percent-progress"`
-		Success         bool   `json:"success"`
-	}
 	for stat := range statuschan {
-		if stat.FileName != tmpfname {
-			fmt.Printf("File: %v\n", stat.FileName)
-			tmpfname = stat.FileName
-		}
-
 		//Raw Input
 		if stat.RawInput != tmprawin {
 			fmt.Printf("Running Command: %v\n", stat.RawInput)
 			tmprawin = stat.RawInput
 		}
-		outline := []byte(stat.RawOutput)
-		if json.Valid(outline) {
-			var output Jsonout
-			_ = json.Unmarshal(outline, &output)
-			if !output.Success {
-				fmt.Printf("%s %s %s Progress:%d/%d(%s) FileKey:%s\r", output.Action.Command, output.Action.File, output.Action.Note,
-					output.ByteProgress, output.TotalSize, output.PercentProgress, output.Action.Key)
-			}
-		} else {
-			ro = stat.RawOutput
-			fmt.Printf("%s", ro)
+		//File Name
+		if stat.FileName != tmpfname {
```

Comment:
Why is this necessary?  We don't want any custom/processed output in the raw output.

Reply:
you mean the  filename printing?

Reply:
Yes

Reply:
you are right